Add SIG Network Spotlight blog article

SD-13 opened the pull request.

SD-13 committed:
doc from Shane

SD-13 committed:
some suggested changes

shaneutt commented on content/en/blog/2023/sig-network-spotlight.md:
```diff
@@ -0,0 +1,144 @@
+implementations (and also provide some areas of alignment so that implementations
+do not become too disparate from each other) upstream Kubernetes efforts are
+underway to create a more modular public interface for `kube-proxy`. The
+intention is for implementations to join in around  a common set of libraries and
```
```suggestion
intention is for implementations to join in around a common set of libraries and
```

SD-13 replied:
done

shaneutt commented on content/en/blog/2023/sig-network-spotlight.md:
```diff
@@ -0,0 +1,144 @@
+
+---
+
+This is a non-exhaustive highlight of sub-projects.
+
+**Sujay**: Couldn’t agree more, I didn’t feel exhausted for a single moment rather it was a very informative description. As you have already mentioned about the sig channels to get involved, would you like to mention anything about where people like beginners can jump in and contribute?
```
Not sure if there is a misunderstanding here, but non-exhaustive in this context wasn't meant to convey a sense or concern for exhaustion: it was meant to point out that there is a lot more going on in SIG Network today and this overview is only "scratching the surface" so to speak.

SD-13 replied:
Honestly, I misunderstood this, thanks for correcting me. I have updated this section a little, PTAL

shaneutt reviewed (approved):
Looks good, I added a couple small comments, one about some blank space and then one about maybe I should use a different phrase than non-exhaustive?

SD-13 committed:
nits

SD-13 commented:
@shaneutt I have addressed your comments but still, if you want to change the phrase `non-exhaustive` to something, please let me know. I think rephrasing is needed there

shaneutt reviewed (approved):
lgtm!

fsmunoz commented:
@SD-13, please `/retitle Add SIG Network Spotlight blog article`

fsmunoz commented on content/en/blog/2023/sig-network-spotlight.md:
```diff
@@ -0,0 +1,144 @@
+author: "Sujay Dey"
+---
+
+Networking is one of the core pillars of Kubernetes and SIG Network, or the Kubernetes Special Interest Group for Networking, is responsible for developing and maintaining the networking features of Kubernetes. It takes care of all aspects to ensure Kubernetes provides a reliable and scalable network infrastructure for containerized applications.
```
It would be more apparent to me if this were phrased like "... and the Special Interest Grup for Networking (SIG Networking)...", initially I got the impression that the "or" was referring to a different group.

SD-13 replied:
done

fsmunoz commented on content/en/blog/2023/sig-network-spotlight.md:
```diff
@@ -0,0 +1,144 @@
+[kong]:https://konghq.com/
+[gwapi]:https://gateway-api.sigs.k8s.io/
+
+**Sujay**: I have to say, that was a really interesting journey! Now let's talk a bit more about sig-network. Since we know that SIG Network covers a lot of grounds, so could you please highlight its scope and current focus areas?
```
sig-network -> SIG Network; likely replace the second reference by "...know that it covers a lot...".

SD-13 replied:
done

fsmunoz commented on content/en/blog/2023/sig-network-spotlight.md:
```diff
@@ -0,0 +1,144 @@
+scope, but I can add some additional highlights on some of our current areas of
+focus:
+
+### KubeProxy / KPNG
```
Using level 3 subheadings seems the most adequate solution, but when rendered it looks like it "breaks" the interview, almost as if it's a separate section to it. The final horizontal ruler tries to overcome this, but it adds an additional separation. Have you tried just using something like `**KubeProxy/KPNG**` in a line by itself? I think that this might make the ruler unneeded. Just a thought though.

SD-13 replied:
Yeah, that makes it more readable, I should have tried this earlier. Done

fsmunoz commented on content/en/blog/2023/sig-network-spotlight.md:
```diff
@@ -0,0 +1,144 @@
+There's a wealth of opportunities here in SIG Network. There are lots of places
+to jump in, and the learning opportunities are boundless.
+
+**Sujay**: Thanks a lot! it was really great discussion with you, I got to know soo many great things about sig-network. I hope many people will find this useful just like I did.
```
sig-network -> SIG Network

SD-13 replied:
done

SD-13 committed:
nits

SD-13 commented:
@fsmunoz I have made the changes, PTAL when you are free. Thanks!
